main.go

```diff
@@ -53,11 +102,28 @@ func getGroupGifs(c *echo.Context) error {
 }
 
 func createGroup(c *echo.Context) error {
-    res := ResponseTemplate{}
-    group := Group{}
+    g := Group{groupSeq, c.Form("name")}
+
```

Comment:
Echo seems to parse the form properly when using `curl -d "name=foo" http://localhost:1323/groups`.

1. Should this API work via image uploading or maybe providing a Gif URL field of some sort?
2. Is Redis good for that kind of storage (images)? Still looking more into how to implement, but will do so in separate branch.